src/envo/envo-edit.owl

```diff
@@ -11827,13 +11827,21 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000532> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000533> (landfill)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000533> "A site at which refuse is dumped, normally filling a natural or artificial depression."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1177%2F0734242X0202000209>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000533> "An anthropogenic contamination feature which is used for solid waste disposal, filling a natural or artificial depression"^^xsd:string)
```

Comment:
MA:ma was an attribution to Micheal Ashburner, who created most of ENVO's early terms.

FFT is the Feature Type Thesaurus, and ADL is the Alexandra Digital Library. These are legacy annotations from ENVO's early days, but likely still valid. 

>Should the wikipedia source (and the general annotation letter codes) be moved to be annotations on the definition?

Yes, sure.

Reply:
>An anthropogenic contamination feature

Not really happy with this superclass, but it should be fine for now.

Reply:
> MA:ma was an attribution to Micheal Ashburner, who created most of ENVO's early terms.
> FFT is the Feature Type Thesaurus, and ADL is the Alexandra Digital Library. These are legacy annotations from ENVO's early days, but likely still valid.

Thanks for the clarification @pbuttigieg  - Would you know if there a document or similar explaining those for ENVO users in general?

Reply:
I would like to have SSSOM files for all our mapped sources (made a ticket for myself here #1129) -- this would make it explicit in the header what the sources are, what the URI expansion is, when the mapping was made.

On the FTT mappings, I made a separate ticket: #1130.